watchfaceloader.qml

```diff
@@ -0,0 +1,37 @@
+import QtQuick 2.9
+
+Item {
+
```

Comment:
Unnecessary empty line :) and I think it's better to give this a different extension, like .qmltemplate because technically speaking this is not valid QML :)